client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -58,17 +55,11 @@ function WPLineItem( {
 	const shouldShowVariantSelector = getItemVariants && item.wpcom_meta && ! isRenewal;
 	const isGSuite = isGSuiteProductSlug( item.wpcom_meta?.product_slug );
 
-<<<<<<< HEAD
 	const productSlug = item.wpcom_meta?.product_slug;
 	const isBusinessPlanProduct = productSlug && isBusinessPlan( productSlug );
```

Comment:
i think it should be safe to remove `productSlug` and `isBusinessPlanProduct` as well.

Reply:
I've had a look at the conflict resolution, looks good to me 👍

Reply:
Nice catch. I have removed them along with a few more unused symbols in f672950.